package.json

```diff
@@ -104,15 +104,15 @@
         },
         {
             "path": "recipe/thirdpartyemailpassword/index.js",
-            "limit": "110kb"
+            "limit": "115kb"
         },
         {
             "path": "recipe/thirdparty/index.js",
             "limit": "100kb"
         },
         {
             "path": "recipe/emailpassword/index.js",
-            "limit": "100kb"
+            "limit": "105kb"
```

Comment:
Size checks are still failing.

Reply:
I updated this (but the sizes didn't change so it will not add a new comment I think)